doc/whats_new.rst

```diff
@@ -196,6 +196,10 @@ Bug fixes
       :clas:`discriminant_analysis.LinearDiscriminantAnalysis` now returns
       correct results. By `JPFrancoia`_
 
+    - Fixed bug in :class:`linear_model.SquaredEpsilonInsensitive`
```

Comment:
+1 for adding tests for each cython class on the gradient values.

the what's new should reference the SGDClassifier as that's what people use
in their code.